pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
 	OpenShiftSharedResourcesNamespace string
 }
 
-func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
+func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
```

Comment:
Can you call them the same things the other ones do?  `fullName` and `name`.  And yes, the others are wrong if they're not like that.